.settings/org.eclipse.m2e.core.prefs

```diff
@@ -0,0 +1,4 @@
+activeProfiles=
```

Comment:
I think that .settings folder should be placed inside .gitignore and dockerignore, and removed from vcs